README.md

```diff
@@ -1,27 +1,113 @@
-# Telemarketing
-Telemarketing is a canonical component that any VTEX app can import.
+# VTEX Telemarketing
+
+## Description
+The VTEX Telemarketing app is a store component that enables a call center operator impersonate a costumer in the store, and this app is used by store theme.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [1.x]    | **Maintenance LTS** |  2018-08-15     | 2018-11-08            | March 2019  | 1.x
 | [2.x]    | **Current Release** |  2018-11-08     |                       |             | 2.x
 
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
-To import it into your code: 
-```js
-import Telemarketing from 'vtex.telemarketing/index'
-```
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+    - [CSS namespaces](#css-namespaces)
+- [Troubleshooting](#troubleshooting)
+- [Tests](#tests)
 
 ## Usage
-You can use it in your code like a React component with the jsx tag: `<Telemarketing />`. 
-```html
-<Telemarketing />
+
+This app uses our store builder with the blocks architecture. To know more about the Store Builder [click here](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object).
+
+We add the telemarketing as a block in our [Store Header](https://github.com/vtex-apps/store-header/blob/master/store/interfaces.json).
+
+To configure or customize this app, you need to import it in your dependencies in `manifest.json`.
+
+```json
+  dependencies: {
+    "vtex.telemarketing": "2.x"
+  }
 ```
 
-DISCLAIMER: This component will only be displayed if the user has the role of `2 - Televendas (Call center operator)` in it´s profile access.
+Then, add `telemarketing` block into your app theme as we do in our [Store theme app](https://github.com/vtex-apps/store-theme/blob/master/store/blocks.json). 
```

Comment:
Now, you can change the behavior of the telemarketing block that is in the store header. See an example of how to configure:
**example**

Reply:
You need to add this, see minicart's example.

Reply:
This app has no behavior to change. So I think it's not necessary to add an example with an empty json.